.github/workflows/e2e.yml

```diff
@@ -40,3 +40,26 @@ jobs:
         name: Tests
         run: |
           bash ./tests/push.sh $IP
+
+  # TODO(milosgajdos): this currently runs E2E tests
+  # against S3 storagedriver only; figure out running E2E tests
+  # against ALL supported cloud storage drivers
+  run-e2e-test-cloud-storage:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Start E2E environment
+        run: |
+          echo '{"insecure-registries" : ["localhost:5000"]}' | sudo tee /etc/docker/daemon.json
```

Comment:
Curious; was this needed? As the daemon should already have 127.0.0.x as insecure by default;

```
Insecure Registries:
  127.0.0.0/8
```

Reply:
Does it have it set on the GHA runner nodes? I think it's done here just in case it doesn't. I can't remember the reason why @wy65701436 added it in originally

Reply:
I don't _expect_ it to be needed, but it's possible GHA runners have a `daemon.json` in place; in that case we should probably check what's in it (as this change would overwrite any other config that's in there; which _could_ be things like a local mirror etc)

Reply:
I mean, ok, but the other e2e tests have been there for couple years now, just for the record.

Now, how do I check what's in `daemon.json`? I don't have access to GH runners 😄 @crazy-max any tips? Should we just create a rando PR and `cat` it? 🤔

Reply:
Ok, made some tests and they pass. I've removed the `daemon.json` updates and the daemon restarts.